shared/features/side-panels/side-panel/side-panel.tsx

```diff
@@ -122,17 +135,39 @@ export const useSidePanel = (
   return useMemo(() => {
     return {
       Panel,
-      open: () => ref.current?.open(config),
-      close: current => ref.current?.close(current),
+      open: (data?: T) => ref.current?.open(data, config),
+      close: (current?: boolean) => ref.current?.close(current),
```

Comment:
Is boolean the right type here?

Reply:
![Capture d’écran 2024-09-11 à 12 08 24](https://github.com/user-attachments/assets/4eada50b-97c7-4c9a-954e-cc011796c630)

yop